core/assets/bundles/bundle.properties

```diff
@@ -1175,6 +1176,7 @@ block.surge-tower.description = An extremely long-range power node with fewer av
 block.diode.description = Battery power can flow through this block in only one direction, but only if the other side has less power stored.
 block.battery.description = Stores power as a buffer in times of surplus energy. Outputs power in times of deficit.
 block.battery-large.description = Stores much more power than a regular battery.
+block.surge-capacitor.description = A massive and expensive battery, the surge capacitor is the ultimate solution to power storage.
```

Comment:
not sure about the usage of `ultimate solution` here, kinda prevents future storage solutions and/or mods 🤔

Reply:
i can think of something else, tbh i didn't think that was a good solution in the first place